content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,185 @@
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+If we now run the code and commit the result to git (and push it to our git
+remote), the new model will show up in the model registry in Studio. You should
+see something like the following picture.
+
+![Newly added model in the Model Registry](/img/mr-newly-added-model.png)
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the Studio's graphical user interface to add
+models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## Versioning models
+
+Now that we have our first model in the model registry, we can start registering
+model versions for the model. We do it by choosing a specific commit in our
+model development history and attaching a version to it to keep an easier track
+of it. We will do that directly in the Studio UI as follows
+
+![Registering model versions](/img/mr-register-model-version.gif)
+
+Since we just saved our model to dvc and added it to the model registry in the
+previous commit, we can just keep the commit which was selected automatically.
+We will also keep the suggested version number. For more details and other way
+of registering model versions you can have a look at the corresponding
+[documentation](/doc/studio/user-guide/model-registry/register-version).
+
+Once we register our first model version, the model registry will also
+automatically connect with the experiment tracking and all metrics which are
+tracked for each model version will also show up in the model registry. We can
+even explore the experiment directly by clicking on the "Open in Project" button
+on the model detail page.
+
+## Assigning lifecycle stages
+
+We have a first version for our model and now it is a good time to assign a
+model lifecycle stage to it. You can create any number of lifecycle stages with
+any names you wish but in this example we will only create two stages called
+"dev" and "staging".
+
+Stages are created whenever a model version is assigned to them. We will now
+assign our model to the "dev" stage as follows.
+
+![Assigning model stages](/img/mr-assign-model-stage.gif)
+
+## Removing stage assignments
```

Comment:
I guess it could even be "changing and removing stages".

Since we allow for multiple stages to be assigned to a model at once, changing them amounts to assigning a new stage and removing the old one. So you kind of learn how to change stages and how to remove them at the same time.